docs/technical_reference/unit_models/steam_ejector.rst

```diff
@@ -0,0 +1,69 @@
+Steam Jet Ejector
+================================
+This Steam Jet Ejector unit model:
+   * Simulates the performance of a steam jet ejector for thermal vapor compression.
+   * Uses semi-empirical correlations for entrainment ratio, PCF, and TCF based on El-Dessouky (1997).
```

Comment:
Should probably spell out the acronyms for PCF and TCF here even though it's explicitly spelled out later in the documentation

Reply:
Should also add a "References" section at the end, which would include the El-Dessouky paper

Reply:
addressed